migrations/slick/src/main/scala/SqlResourceMigration.scala

```diff
@@ -0,0 +1,23 @@
+package com.liyaos.forklift.slick
+
+import com.liyaos.forklift.core.Migration
+import slick.dbio.DBIO
+import slick.jdbc.JdbcProfile
+
+import scala.io.Source
+
+trait SqlResourceMigrationInterface[T] extends Migration[T, DBIO[Unit]] {
+  def sqlResource: String
+}
+
+case class SqlResourceMigration[T](id: T, profile: JdbcProfile, clazz: Class[_]) extends SqlResourceMigrationInterface[T] {
+
+  import profile.api._
+
+  val sqlResource = Source.fromInputStream(clazz.getResourceAsStream(s"$id.sql")).mkString
```

Comment:
I have never tried this, but it looks like you can do `Source.fromResource` since Scala 2.12?

Reply:
This is minor, but `sqlQueries` may be a more suitable name here?

Reply:
Will make those changes

Reply:
`Source.fromResource` turns out to be much cleaner, since you no longer have to pass in a class to figure out the resource location from.